pvlib/singlediode.py

```diff
@@ -247,6 +250,10 @@ def bishop88_i_from_v(voltage, photocurrent, saturation_current,
     method : str, default 'newton'
        Either ``'newton'`` or ``'brentq'``. ''method'' must be ``'newton'``
        if ``breakdown_factor`` is not 0.
+    kwargs : keyword arguments
```

Comment:
I think this should this be

```
kwargs : dict-like
```

Reply:
Oh, that's true. Other functions have this: `kwargs : dict`. Since `dict-like` would allow for `pd.Series`, I will keep the former.